domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -231,6 +231,28 @@ class StoryProgressTestHelper @Inject constructor(
     )
   }
 
+  /**
+   * Marks full topic progress on the second test topic for a particular profile.
+   *
+   * @param profileId The profile we are setting topic progress for.
+   * @param timestampOlderThanOneWeek If the timestamp for completing the topic is from more than
+   *     one week ago.
```

Comment:
This can go in one line as ktlint 100 char length check is not on comments.

Reply:
Oh, got it, thanks.